packages/tx-construction/test/tx-builder/TxBuilder.test.ts

```diff
@@ -349,6 +405,25 @@ describe('GenericTxBuilder', () => {
       it('legit output with valid with address and coin', async () => {
         await expect(txBuilder.buildOutput().address(address).coin(output1Coin).build()).resolves.toBeTruthy();
       });
+
+      it('resolves handle to address', async () => {
+        const txOut = await txBuilder.buildOutput().handle('alice').coin(output1Coin).build();
+
+        expect(txOut.handle).toBe(resolvedHandle);
+        expect(txOut.address).toBe(resolvedHandle.resolvedAddresses.cardano);
+      });
+
+      it('rejects with an error when a handle is not found', async () => {
+        await expect(
+          txBuilderWithNullHandles.buildOutput().handle('alice').coin(output1Coin).build()
+        ).rejects.toThrowError(OutputValidationMissingRequiredError);
+      });
+
+      it('rejects with an error when handle is not found', async () => {
```

Comment:
```suggestion
      it('rejects with an error when handle provider fails to resolve', async () => {
```

Reply:
aa5eaf7df693d418c3cdb800b0e524e2e316ae28 should fix this